var/spack/repos/builtin/packages/mbdyn/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
Can you update the copyright to `2013-2020`?

Reply:
Fixed.